src/Model.php

```diff
@@ -1684,6 +1684,14 @@ public function export(array $fields = null, $key_field = null, $typecast_data =
         return $res;
     }
 
+    /**
+     * Number of records in current model scope
+     */
+    public function getCount(): int
```

Comment:
It is not get... I think we should use something like `toQuery(Query::TYPE_COUNT)->getOne()`.